dev-docs/bidder-adaptor.md

```diff
@@ -288,6 +292,7 @@ Notes on parameters in the bidderRequest object:
 - **refererInfo** is provided so you don't have to call any utils functions. See below for more information.
 - **gdprConsent** is the object containing data from the [GDPR ConsentManagement](/dev-docs/modules/consentManagement.html) module. For TCF2+, it will contain both the tcfString and the addtlConsent string if the CMP sets the latter as part of the TCData object.
 - **uspConsent** is the object containing data from the [US Privacy ConsentManagement](/dev-docs/modules/consentManagementUsp.html) module
+- **ortb2** is the global (not specific to any adUnit) [first party data](/features/firstPartyData.html) to use for all requests in this auction.
```

Comment:
It might also include other standard ortb2 fields and extensions that might not typically be considered first party data, such as user agent client hints (device.sua) or information on applicable regulations eg ( regs.ext.gpc )

Reply:
Updated, but I'll accept suggestions on how to word it less clunkily.